sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DocumentCollection.java

```diff
@@ -427,12 +430,18 @@ public DocumentCollection getWrappedItem() {
 
         private void writeObject(ObjectOutputStream objectOutputStream) throws IOException {
             documentCollection.populatePropertyBag();
-            objectOutputStream.writeObject(documentCollection.getPropertyBag());
+            ObjectNode docCollectionNode = OBJECT_MAPPER.createObjectNode();
+            docCollectionNode.set("col", documentCollection.getPropertyBag());
```

Comment:
is "col" part of the service side contract?

Reply:
No - just some random json property for the collection object (in case we need to push more fields than just altLink)